chart/openfaas/templates/gateway-dep.yaml

```diff
@@ -137,6 +137,12 @@ spec:
           value: "{{ .Values.nats.channel }}"
         {{- end }}
         {{- end }}
+        {{- if or .Values.nats.external.enabled (eq .Values.queueMode "jetstream") }}
+        {{- if eq .Values.queueMode "jetstream" }}
+        - name: faas_nats_jetstream
```

Comment:
Isn't this now just `nats_jetstream`?

Why are you also looking at `.Values.nats.external.enabled`?

Reply:
> Isn't this now just `nats_jetstream`?

Yes it is renamed now but we still need a new release of the gateway before I can update it here or it will break the chart for people that are testing with JetStream.

> Why are you also looking at `.Values.nats.external.enabled`?
Good question I was wondering the same thing now that I look back at this. I think it needs to be removed because it will break running with external nats streaming.